### PR TITLE
Changes I had to make to make evaluation works

### PR DESCRIPTION
**Pull request by JoanFM**
Hello guys,

Here there are some changes I needed to make the evaluation work under python3.7, maybe you find it helpful

### CODE REVIEW

**Comment by JoanFM** [comment]
It is also important to note, that the files from the documentation do not actually contain the images (the images folder is just a symlink to nowhere ;)), but they are available as a `kaggle` dataset.

File: model.py [hunk 1]
```diff
@@ -361,7 +363,6 @@ def forward_loss(self, img_emb, cap_emb, **kwargs):
         """Compute the loss given pairs of image and caption embeddings
         """
         loss = self.criterion(img_emb, cap_emb)
-        self.logger.update('Le', loss.data[0], img_emb.size(0))
```
Comment (fartashf) [inline model.py:364]: Can you comment on why this line is removed? Thanks.

**Review by fartashf — COMMENTED** [review]
Thanks for the PR. Do you have new links for images?

File: model.py (hunk 1 above)
Comment (JoanFM) [reply model.py:364]: I am not sure at this moment, I guess there is no need for it

**Comment by JoanFM** [comment]
> Thanks for the PR. Do you have new links for images?

I just extracted them from `kaggle` dataset.

File: model.py (hunk 1 above)
Comment (fartashf) [reply model.py:364]: This line is important for analyzing the behaviour of training as it logs the loss. Would you be able to add it back? The rest of the PR looks good to me. Thanks.

Comment (JoanFM) [reply model.py]: Done!

**Review by fartashf — APPROVED** [review]
